From 7218b9ad08e6d0264a314151b840be0e997ed688 Mon Sep 17 00:00:00 2001
From: Vassil Vassilev <v.g.vassilev@gmail.com>
Date: Sat, 30 Mar 2024 17:06:30 +0000
Subject: [PATCH] [cmake] Do not build the error estimation codes with
 differentiable clang.

We do not need to use a clad-compatible clang to build the error estimation
demos.
---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e8e1900e..934dbe5c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -317,6 +317,9 @@ if (NOT CLAD_BUILD_STATIC_ONLY)
     include(GoogleBenchmark)
   endif(CLAD_ENABLE_BENCHMARKS)
 
+  add_subdirectory(demos/ErrorEstimation/CustomModel)
+  add_subdirectory(demos/ErrorEstimation/PrintModel)
+
   # Change the default compiler to the clang which we run clad upon. Our unittests
   # need to use a supported by clad compiler. Note that's a huge hack and it is
   # not guaranteed to work with cmake.
@@ -330,8 +333,6 @@ if (NOT CLAD_BUILD_STATIC_ONLY)
 
   add_subdirectory(unittests)
   add_subdirectory(test)
-  add_subdirectory(demos/ErrorEstimation/CustomModel)
-  add_subdirectory(demos/ErrorEstimation/PrintModel)
 
   # Add benchmarking infrastructure.
   if (CLAD_ENABLE_BENCHMARKS)
